### PR TITLE
making singlepage call to remote by default

### PR DESCRIPTION
**Pull request by prklm10**
Taking singlepage ss for app automate from mobile.
Adding `PERCY_DISABLE_REMOTE_UPLOADS` env variable to override above.
Adding `PERCY_ENABLE_DEV` for dev. Added it so we can test better with sdk.

### CODE REVIEW

File: test/percy/providers/appAutomateProvider.test.mjs [hunk 1]
```diff
@@ -104,4 +104,40 @@ describe('AppAutomateProvider', () => {
       expect(appAutomate.debugUrl).toEqual(null);
     });
   });
+
+  describe('getTiles', () => {
+
+    it('when env isDisableRemoteUpload is true', async () => {
```
[the remark this answers is not in the document]
Comment (prklm10) [reply test/percy/providers/appAutomateProvider.test.mjs]: - Updated tests.
- For fullpage
     -  If the env variable  isDisableRemoteUpload is set then it should take only singlepage ss with local appium right? Or do we take fullpage regardless?

[the remark this answers is not in the document]
Comment (ninadbstack) [reply test/percy/providers/appAutomateProvider.test.mjs]: We should take just a single page, but we should ideally print a warning stating `Full page screenshots are only supported when `isDisableRemoteUpload` is not set.`

File: percy/providers/appAutomateProvider.js [hunk 2]
```diff
@@ -95,17 +95,26 @@ class AppAutomateProvider extends GenericProvider {
...
     }
 
+    let screenshotType = 'fullpage';
+    let projectId = 'percy-prod';
+    if (fullPage !== true) {
+      screenshotType = 'singlepage';
+    }
+    if (this.isPercyDev()) {
+      projectId = 'percy-dev';
+    }
```
Comment (ninadbstack) [inline percy/providers/appAutomateProvider.js:110]: Test for this is missing

Comment (prklm10) [reply percy/providers/appAutomateProvider.js:113]: I had added tests within `getTiles` describe. I added separate describe for the tests.

**Review by ninadbstack — APPROVED** [review]

**Pull request merged**